COVID_19_model.ipynb

```diff
@@ -1652,8 +1674,140 @@
+        "                  yaxis=go.layout.YAxis(title=go.layout.yaxis.Title(\n",
+        "                      text=\"Death count\",\n",
+        "                      font=dict(size=axis_label_font_size))),\n",
+        "                  font=dict(size=tick_font_size))\n",
+        "  # fig.update_yaxes(range=[0, 1000])\n",
+        "\n",
+        "  fig.show()\n",
+        "\n",
+        "  plot_extension = \"_cumulative\" if cumulative else \"\"\n",
+        "  plotly_offline_nojs(fig, filename='forecast_plots/single_plot_combined' + (\n",
+        "      plot_extension) + '.html')"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "dGYRrUVb4aV3",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "#@title Combined grid plot\n",
+        "for cumulative in [False, True]:\n",
+        "  plot_countries = countries_plot_data.keys()\n",
+        "  num_plot_countries = len(plot_countries)\n",
+        "  num_cols = 2\n",
+        "  num_rows = int(np.ceil(num_plot_countries / num_cols))\n",
+        "  fig = plotly.subplots.make_subplots(rows=num_rows, cols=num_cols,\n",
+        "                                      subplot_titles=tuple(plot_countries))\n",
+        "\n",
+        "  for i, country in enumerate(plot_countries):\n",
+        "    print(\"Adding traces for country {} of {}\".format(i+1, num_plot_countries))\n",
+        "    country_plot_data = countries_plot_data[country]\n",
+        "    simple_model_data = analysis_df.iloc[(\n",
+        "      analysis_df[[country_col_name]].values[:, 0] == country)]\n",
+        "\n",
+        "    # pdb.set_trace()\n",
+        "    traces = plot_country_posterior(\n",
+        "      country,\n",
+        "      country_plot_data.posterior_samples,\n",
+        "      simple_model_data,\n",
+        "      save_plot=False,\n",
+        "      pred_vals=country_plot_data.pred_vals,\n",
+        "      cumulative=cumulative,\n",
+        "      show_plot=False,\n",
+        "    )\n",
+        "\n",
+        "    # Add all traces to the correct subplot\n",
+        "    row = 1 + i // num_cols\n",
+        "    col = 1 + i % num_cols\n",
+        "    for j, t in enumerate(traces):\n",
+        "      if j == len(traces)-2:\n",
+        "        name = \"Forecast\"\n",
+        "      elif j == len(traces)-1:\n",
+        "        name = \"Actual deaths\"\n",
+        "      else:\n",
+        "        name = \"Posterior samples\"\n",
+        "      t.legendgroup = name\n",
+        "      t.name = name\n",
+        "      t.showlegend = (i==0 and j >= (len(traces)-3))\n",
+        "      fig.add_trace(t, row=row, col=col)\n",
+        "\n",
+        "  title_text = \"Cumulative deaths forecast\" if cumulative else (\n",
+        "      \"Daily deaths forecast\")\n",
+        "  fig.update_layout(height=300*num_rows, title_text=title_text, title_x=0.5,\n",
```

Comment:
what's the height for? also the font size is fixed and not defined with the others

Reply:
I suggest we just merge this and fix any remaining issues in a subsequent PR.

Reply:
Sure